app/assets/javascripts/qpixel_api.js

```diff
@@ -263,6 +263,95 @@ window.QPixel = {
     }
   },
 
+  filters: async () => {
+    if (this._filters == null && localStorage['qpixel.user_filters']) {
+      this._filters = JSON.parse(localStorage['qpixel.user_filters']);
+    }
+    else if (this._filters == null) {
+      // If they're still null (or undefined) after loading from localStorage, we're probably on a site we haven't
+      // loaded them for yet. Load via AJAX.
+      const resp = await fetch('/users/me/filters', {
+        credentials: 'include',
+        headers: {
+          'Accept': 'application/json'
+        }
+      });
+      const data = await resp.json();
+      localStorage['qpixel.user_filters'] = JSON.stringify(data);
+      this._filters = data;
+    }
+
+    return this._filters;
+  },
+
+  defaultFilter: async categoryId => {
+    const resp = await fetch(`/users/me/filters/default?category=${categoryId}`, {
+      credentials: 'include',
+      headers: {
+        'Accept': 'application/json'
+      }
+    });
+    const data = await resp.json();
+    return data.name;
+  },
+
+  setFilterAsDefault: async (category_id, name) => {
```

Comment:
request (style): there are a few variables in snake_case in this file - unless there's a reason they need to be, can we change to camelCase?

Reply:
Will do 👍